scripts/build_demand_profiles.py

```diff
@@ -181,8 +181,19 @@ def build_demand_profiles(
     gegis_load = gegis_load.to_dataframe().reset_index().set_index("time")
     # filter load for analysed countries
     gegis_load = gegis_load.loc[gegis_load.region_code.isin(countries)]
-    logger.info(f"Load data scaled with scaling factor {scale}.")
-    gegis_load["Electricity demand"] *= scale
+
+    if scale is not None:
```

Comment:
Great that you have thought about edge cases! However, not sure this particular check makes sense, as if `scale` is not defined it would cause an error earlier, at https://github.com/pypsa-meets-earth/pypsa-earth/blob/d89a9a384ccff8b65adba6cae835a6df6ee53334/scripts/build_demand_profiles.py#L249 So, I'd remove this check which would also make the code easier to read. What do you think?

Reply:
I agree